File: examples/holidays/list/node/getHolidays.js
const { HolidaysService } = require("m3o/holidays");

// List the holiday dates for a given country and year
async function getHolidays() {
  let holidaysService = new HolidaysService(process.env.MICRO_API_TOKEN);
  let rsp = await holidaysService.list({
    country_code: "GB",
    year: 2022,
  });
  console.log(rsp);
}

getHolidays();

File: examples/holidays/list/go/getHolidays.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/holidays"
)

// List the holiday dates for a given country and year
func GetHolidays() {
	holidaysService := holidays.NewHolidaysService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := holidaysService.List(&holidays.ListRequest{
		Year: 2022,
	})
	fmt.Println(rsp, err)
}
